src/main/java/com/palettee/user/domain/MajorJobGroup.java

```diff
@@ -22,7 +23,22 @@ public static MajorJobGroup findMajorGroup(String input) {
                 .orElse(null);
     }
 
-    public String getMajorGroup() {
+    public static MajorJobGroup of(String group) throws InvalidJobGroupException {
+        String upper = group.toUpperCase();
+
+        MajorJobGroup majorGroup = Arrays.stream(MajorJobGroup.values())
+                .filter(job -> job.toString().equals(upper))
+                .findFirst()
+                .orElse(null);
+
+        if (majorGroup == null) {
+            throw InvalidJobGroupException.EXCEPTION;
+        }
```

Comment:
```suggestion
        MajorJobGroup majorGroup = Arrays.stream(MajorJobGroup.values())
                .filter(job -> job.toString().equals(upper))
                .findFirst()
                .orElseThrow(() -> InvalidJobGroupException.EXCEPTION);
```
이렇게 수정할 수 있을 것 같습니다!

Reply:
https://github.com/prgrms-web-devcourse-final-project/WEB1_1_ZeroOne_BE/pull/39/commits/6562876cf39107cbe0d6d3645846aa7a427eee2e 로 개선하였습니다!
`MiniorJobGroup` 쪽도 동일하게 변경하였습니다.